projects/metis/src/app/dataset/workflow/workflow.component.ts

```diff
@@ -498,10 +523,11 @@ export class WorkflowComponent extends SubscriptionManager implements OnInit {
           plugins.push(dataHTTP);
           plugins.push(dataOAIPMH);
         } else {
-          conf.parameterFields = conf.parameterFields ? conf.parameterFields : [];
-          plugins.push(
-            this.formatFormValue(conf.label as PluginType, conf.parameterFields, enabled)
-          );
+          let parameterFields: ParameterField = [];
+          if (conf.parameterFields) {
+            parameterFields = conf.parameterFields;
+          }
+          plugins.push(this.formatFormValue(conf.label as PluginType, parameterFields, enabled));
```

Comment:
The old version had the side effect of ensuring that `conf.parameterFields` is always defined. Is that effect no longer needed?

Reply:
According to my testing the effect was never needed!

Reply:
It would seem not to be a dangerous change.